assets/tests/components/detours/diversionPageActivate.test.tsx

```diff
@@ -81,4 +81,14 @@ describe("DiversionPage activate workflow", () => {
 
     expect(activateDetourButton.get()).toBeVisible()
   })
+
+  test("clicking the activate button shows the 'Active Detour' screen", async () => {
```

Comment:
non-blocking: there is not yet an "active detour" screen, so the test (appropriately, I think) checks that the old screen is gone. Should the name of the test be changed?
```suggestion
  test("clicking the activate button changes screen from 'Share Detour Details'", async () => {
```

Reply:
Errr - woops! Auto-merge scooped your (very accurate) comment.

But I think https://github.com/mbta/skate/pull/2715 makes the test name accurate again.